blog post v13

ntarocco opened the pull request.

ntarocco commented on content/blog/2025-07-22-invenio-rdm-13/contents.lr:
```diff
@@ -0,0 +1,201 @@
+
+## Release Notes
+
+See the full release notes at [https://inveniordm.docs.cern.ch/releases/v13/version-v13.0.0/](https://inveniordm.docs.cern.ch/releases/v13/version-v13.0.0/) and the upgrade guide at [https://inveniordm.docs.cern.ch/releases/v13/upgrade-v13.0/](https://inveniordm.docs.cern.ch/releases/v13/upgrade-v13.0/).
```
Done!

ntarocco commented on content/blog/2025-07-22-invenio-rdm-13/contents.lr:
```diff
@@ -0,0 +1,201 @@
+InvenioRDM v13.0 focuses on enhancing **administrative capabilities**, improving **community features**, and introducing powerful new tools for **data management and organization**.
+Here are some of the **major highlights**, be sure to check the release notes for a comprehensive list of new features and improvements.
+
+### Audit Logs
```
Done! I have a small doubts on Jobs (looks to me that it was a very much wanted feature, but I am perfectly fine to keep it there)

mfenner commented:
@ntarocco I think the screenshot looks cut off because of the following in style.css:

```
.blog-list-item-content {
  height: 250px;
  ...
```

ntarocco commented:
In the end I have changed screenshot, it looks better now. I have also added a small shadow at the bottom of each picture in all blog post, let me know if you don't like it.

![Screenshot 2025-07-29 at 09 27 28](https://github.com/user-attachments/assets/c9e14c46-579f-44a2-a823-c94a4a4bbd9a)

mfenner commented:
Published blog post looks good!
